tests/scripts/thread-cert/Cert_5_3_10_AddressQuery.py

```diff
@@ -172,7 +172,7 @@ def test(self):
 
         # Send an ICMPv6 Echo Request from MED1 to ROUTER1 GUA PREFIX_1 address
         self.assertFalse(self.nodes[MED1].ping(router1_addr))
-        self.simulator.go(1)
+        self.simulator.go(3)
```

Comment:
The problem you identified in https://github.com/openthread/openthread/issues/6018#issuecomment-752850508 seems to indicate we need to add more time to the 580s. Is this still true?

Reply:
The failed packets show there is Router_1 ID in the leader's advertisements after Router_1 power off 580,  add routerselectionjitter=1 for Leader

Reply:
Why does setting the leader's `router_selection_jitter` to 1 resolve this issue?